azure-container-apps/ReadMe.md

```diff
@@ -11,146 +11,20 @@ The deployment consists of two [containers](https://learn.microsoft.com/en-us/az
 
 **Table of contents:**
 
-- [Overview](#Overview)
-- [Prerequisites](#Prerequisites)
-- [Getting Started: Generate credentials for automated user provisioning with 1Password](#Generate-credentials-for-automated-user-provisioning-with-1Password)
-- [Azure Container App Deployment steps using the Azure Shell](#azure-container-app-deployment-steps-using-the-azure-shell)
+- [Standard Deployment](#standard-deployment)
 - [Update your SCIM bridge](#update-1password-scim-bridge)
```

Comment:
Maybe we can put all the below list items in an Appendix and link to that section from the "advanced references" link in `\README.md`?

Reply:
I think we should review this during Jacob's review on the best way to present this info.